src/plugins/explore/public/components/chart/chart.tsx

```diff
@@ -45,11 +43,13 @@ export const DiscoverChart = ({
   showHistogram,
 }: DiscoverChartProps) => {
   const { from, to } = data.query.timefilter.timefilter.getTime();
-  const timeRange = {
-    from: dateMath.parse(from)?.format('YYYY-MM-DDTHH:mm:ss.SSSZ') || '',
-    to: dateMath.parse(to, { roundUp: true })?.format('YYYY-MM-DDTHH:mm:ss.SSSZ') || '',
-  };
-  const { interval } = useSelector((state) => state.legacy);
+  const timeRange = useMemo(() => {
+    return {
+      from: dateMath.parse(from)?.format('YYYY-MM-DDTHH:mm:ss.SSSZ') || '',
+      to: dateMath.parse(to, { roundUp: true })?.format('YYYY-MM-DDTHH:mm:ss.SSSZ') || '',
+    };
+  }, [from, to]);
```

Comment:
why would the results be different with useMemo, if it's computing from `from` and `to` either ways?